docs/webhooks.rst

```diff
@@ -93,6 +93,44 @@ GitLab
 * Leave the default **Push events** selected and mark **Tag push events** also
 * Finish by clicking **Add Webhook**
 
+Gitea
+~~~~~
+
+These instructions apply to any Gitea instance.
+
+.. warning::
+
+   This isn't officially supported, but using the "GitHub webhook" is an effective workaround,
+   because Gitea uses the same payload as GitHub. The generic webhook is not compatibile with Gitea.
+   See `issue #8364`_ for more details. Official support may be implemented in the future.
```

Comment:
> Official support may be implemented in the future.

I'm not sure that decision has been made.

Reply:
"may" or "may not", I don't think this paragraph conveys any certain decision?

Reply:
it gives a "false hope", I don't see us adding another service like gitea, but implementing another solution like having a more configurable webhook processing instead.

Reply:
I'm sorry the sentence sounds as a promise, it wasn't meant to be. I was trying to address [this comment](https://github.com/readthedocs/readthedocs.org/pull/8402#issuecomment-896692389) by @humitos. Feel free to remove it or rephrase it of course!

Reply:
no worries, this might need some input from @ericholscher or @agjohnson if we actually want to add this to our roadmap.